sentence_encoders/02_extract_all_encoded_sentences.ipynb

```diff
@@ -0,0 +1,1363 @@
+{
```

Comment:
<p>Interesting that the encoder sees 'noncoding' as a verb instead of an adjective so it converts it to 'noncode'. Doesn't really matter, I just hadn't seen a encoder run into an unfamiliar word like that before </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/snorkeling-full-text/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Interesting point; however, that is actually due to Spacy's parser. The lemmatizer for Spacy recognized that word has an 'ing' and assumed it was a verb so it auto changed it to noncode. 🤷🏾‍♂️ as Spacy quite good for English but half and half for science devised words.